New alien factions: the Builders and the Ka'het (#4716)
endless-sky/endless-sky: data/ka'het missions.txt

				decline
	on decline
		log "Asked Taely about the Graveyard and the Ka'het. She spoke of the previous contacts between them and the Remnant."
		log "Factions" "Ka'het" `The Ka'het are aliens which live in an area south of the Ember Waste, called the Graveyard. They are large lifeforms which can travel in space using artificial exoskeletons and will attack anyone on sight, even sometimes other Ka'het.`

mission "First Contact: Ka'het: Quarg 1"
	name "Abandoned Ringworld"
	description "Visit a Quarg station to ask them about the abandoned ring in the middle of Ka'het space."
	landing
	source "Querii Ka"
	destination "Lagrange"

> 		has "First Contact: Quarg: offered"

Reviewer: Missing the `to offer` for this. I question if we want to prevent this mission from offering if the player hasn't already met the Hai, though. The Quarg first contact mission makes it clear that the player already knows what the Quarg are and what their ringworlds are, so I imagine the player would recognize the ringworld in the Graveyard and know that something is up since it's deserted. You'd just need to add a `branch` in the conversation based off of if the player `has "First Contact: Quarg: offered"` to change it.

# Copyright (c) 2019 by Tommaso Becca
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

mission "Graveyard Label"
	landing
	invisible
	source
		attributes ka'het
	on offer
		event "graveyard label"
		fail

event "graveyard label"
	galaxy "label graveyard"
		sprite label/graveyard

mission "Disabled Ka'het"
	boarding
	invisible
	source
		government "Ka'het" " Ka'het "
	destination "Earth"
	on offer

> 		log "Successfully boarded a Ka'het, it appears to be a single, ship-sized organism that can travel in space using a special exoskeleton."

Reviewer: ```suggestion
		log "Successfully boarded a Ka'het. It appears to be a single, ship-sized organism that can travel in space using a special exoskeleton."
```

	on offer
		conversation
			`Do you want to ask the Remnant about the section of space south of the Ember Wastes, the Graveyard, and its inhabitants?`
			choice
				`	(Yes.)`
				`	(Not now.)`
					defer
			`	You ask around more than a dozen people, but none of them seems to be aware of what you're talking about. Either the information is too confidential to be given to a stranger like you, or you didn't ask the right people.`
				decline

mission "First Contact: Ka'het: Remnant 1B"
	landing
	invisible
	source "Viminal"
	to offer
		has "Graveyard Label: offered"
		has "remnant met taely"
	on offer
		conversation
			`Do you want to ask Taely about the section of space south of the Ember Wastes, the Graveyard, and its inhabitants?`
			choice
				`	(Yes.)`
				`	(Not now.)`
					defer
			`	You find Taely inside one of the outer hangars of the shipyard, apparently teaching a small group of people how to make basic repairs on an Albatross. You sit on a bench waiting for her to finish her lesson, after which she walks over to you. "Hello, <first>!" she says. "Is there something I can help you with?" `
			`	"I'm here because I need to ask you a few questions," you reply.`

> 			`	As she sits near you, you start explaining her about the area you reached, and the weird alien ships that attacked you with weapons similar to the ones the Remnant possess. It doesn't seem to be news for her. "We know of the existence of these Ka'het, the aliens you encountered, since we first reached the Ember Wastes. We found many of their 'shells' in our planets, crashed there thousands or even tens of thousands of years before our arrival, but with some tech so advanced to still be conserved enough to be reverse engineered.`

Reviewer: ```suggestion
			`	As she sits near you, you start explaining to her the area you reached, including the weird alien ships that attacked you with weapons similar to the ones the Remnant possess. It doesn't seem to be news for her. "We know of the existence of these Ka'het, the aliens you encountered, since we first reached the Ember Wastes. We found many of their 'shells' in our planets, crashed there thousands or even tens of thousands of years before our arrival, but with some tech so advanced to still be conserved enough to be reverse engineered.`
```
Technically the player could get this mission without having been attacked, but I'll let it slide.

mission "First Contact: Ka'het: Quarg 1"
	name "Abandoned Ringworld"
	description "Visit a Quarg station to ask them about the abandoned ring in the middle of Ka'het space."
	landing
	source "Querii Ka"
	destination "Lagrange"
		has "First Contact: Quarg: offered"
	on offer
		log "Discovered an uninhabited, complete Quarg ring in the middle of the Graveyard. It seems weird that they would abandon a ringworld, given how important they are to them."
		conversation
			`Due to your past experience, you expect crowds of the tall and inexpressive Quarg walking near the landing pads of this ringworld; instead, the whole station seems uninhabited.`
			`	Apart from the utter silence, which is pretty common in most rings, you are convinced by the fact that every room, apartment and cabin has been carefully emptied, leaving nothing behind. Perhaps it would be wise to ask the Quarg what happened here...`
				accept

mission "First Contact: Ka'het: Quarg 2"
	landing
	invisible
	source

> 		planet "Lagrange"

Reviewer: This can just be `source "Lagrange"`.

				decline
	on decline
		log "Asked Taely about the Graveyard and the Ka'het. She spoke of the previous contacts between them and the Remnant."
		log "Factions" "Ka'het" `The Ka'het are aliens which live in an area south of the Ember Waste, called the Graveyard. They are large lifeforms which can travel in space using artificial exoskeletons and will attack anyone on sight, even sometimes other Ka'het.`

mission "First Contact: Ka'het: Quarg 1"
	name "Abandoned Ringworld"
	description "Visit a Quarg station to ask them about the abandoned ring in the middle of Ka'het space."
	landing

> 	source "Querii Ka"

Reviewer: ```suggestion
	source "Querii'ka"
```

# Copyright (c) 2019 by Tommaso Becca
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

mission "Graveyard Label"
	landing
	invisible
	source
		attributes ka'het
	on offer
		event "graveyard label"
		fail

event "graveyard label"
	galaxy "label graveyard"
		sprite label/graveyard

mission "Disabled Ka'het"
	boarding
	invisible
	source
		government "Ka'het" " Ka'het "
	destination "Earth"
	on offer

> 		log "Successfully boarded a Ka'het, it appears to be possible by killing the organism without blowing up its artificial exoskeleton."

Reviewer: I feel like instead of this nonchalant description of how to board a Ka'het by just not killing the internal organism, this log should instead focus on the fact that there is some sort of organism inside of an exoskeleton. It shouldn't be "lol found out how to board one," it should be "wtf this thing is alive."

Author: Changed to the latter

		conversation

> 			`Your ship slowly approaches the <origin>, now floating lifeless near you. As you move closer, you start noticing many holes in the weakest points of the hull and damages reaching what appears to be a large, slug-like being in the middle of the ship. While most of the ship's machinery doesn't seem to have received heavy damages, your weapons must have hit the slug's vital organs, killing it.`

Reviewer: ```suggestion
			`Your ship slowly approaches the <origin>, now floating lifeless near you. As you move closer, you spot a massive hole in the hull of the ship, revealing what appears to be a large, slug-like being. While most of the ship's machinery doesn't seem to have received heavy damages, your weapons must have hit the slug's vital organs, killing it, after which point the ship seems to have stopped working.`
```

			`	You have no troubles docking with what seems to be its "exoskeleton", trying to decide what to do next: while the Ka'het is too different from a ship for you to command it, it's safe to say that you can now loot its outfits.`
				decline

mission "First Contact: Ka'het: Remnant 1A"
	landing

> 	minor

Reviewer: `minor` is only used for spaceport missions, and is therefore unnecessary here.

	invisible
	source
		government "Remnant"
	to offer
		has "Graveyard Label: offered"

> 		"remnant met taely" == 0

Reviewer: ```suggestion
		not "remnant met taely"
```

> 		not "Remnant: Technology Available: offered"

Reviewer: This condition doesn't seem necessary given that the second mission isn't also checking that you do have this mission offered.

	on offer
		conversation
			`Do you want to ask the Remnant about the section of space south of the Ember Wastes, the Graveyard, and its inhabitants?`
			choice
				`	(Yes.)`
				`	(Not now.)`
					defer
			`	You ask around more than a dozen people, but none of them seems to be aware of what you're talking about. Either the information is too confidential to be given to a stranger like you, or you didn't ask the right people.`
				decline

mission "First Contact: Ka'het: Remnant 1B"
	landing
	minor
	invisible
	source "Viminal"
	to offer
		has "Graveyard Label: offered"

> 		"remnant met taely" >= 1

Reviewer: ```suggestion
		has "remnant met taely"
```

	on offer
		conversation
			`Do you want to ask Taely about the section of space south of the Ember Wastes, the Graveyard, and its inhabitants?`
			choice
				`	(Yes.)`
				`	(Not now.)`
					defer

> 			`	You find Taely inside one of the outern hangars of the shipyard, apparently teaching a small group of people how to operate the basic repairs over an Albatross. You sit on a bench waiting for her to finish her lesson, and in a few minutes you see her walking towards you. "Hello, <first>!" she says. "Sorry, but I have nothing for you to do today." "Actually," you reply, "I'm here because I need to make you a few questions."`

Reviewer: ```suggestion
			`	You find Taely inside one of the outer hangars of the shipyard, apparently teaching a small group of people how to make basic repairs on an Albatross. You sit on a bench waiting for her to finish her lesson, after which she walks over to you. "Hello, <first>!" she says. "Is there something I can help you with?" `
			`	"I'm here because I need to ask you a few questions," you reply.`
```

> 			`	As she sits near you, you start explaining her about the area you reached, and the weird alien ships that attacked you with weapons similar to the ones the Remnant possess. It doesn't seem to be news for her. "We know of the existance of these Ka'het, the aliens you encountered, since we first reached the Ember Wastes. We found many of their 'shells' in our planets, crashed there thousands or even tens of thousands years before our arrival, but with some tech so advanced to still be conserved enough to be reverse engineered.`

Reviewer: ```suggestion
			`	As she sits near you, you start explaining her about the area you reached, and the weird alien ships that attacked you with weapons similar to the ones the Remnant possess. It doesn't seem to be news for her. "We know of the existence of these Ka'het, the aliens you encountered, since we first reached the Ember Wastes. We found many of their 'shells' in our planets, crashed there thousands or even tens of thousands of years before our arrival, but with some tech so advanced to still be conserved enough to be reverse engineered.`
```

> but with some tech so advanced to still be conserved enough to be reverse engineered.

This is a perfect example of what I am intending when I say that the Builders could have left behind massive pristine cities. If the Ka'het technology is able to survive this long in such a good condition, why not their cities?

					goto shells
			label tech
			`	"Back then we found large numbers of their EMP torpedoes, most of which in almost pristine conditions; their internal mechanisms were also simple enough we were able to reproduce them perfectly. We didn't find enough of the other systems in the 'shells' to do the same, but their tech still have been our main inspiration point for centuries."`
			choice
				`	"Wait, why did you call them 'shells' and not ships?"`
			label shells
			`	"You see, <first>," she says, "the Ka'het aren't normal spaceships. The debris we found were artificial exoskeletons, built around huge living beings; it's unlikely a species that big could have developed that sort of technology, so we presume someone must have created it for them. This could have happened in the Graveyard, since it's the only place where we know them to still live in, but we have such a limited data about that section of space that we can't say for sure." As she finishes talking, one of the people Taely's was talking with earlier comes closer and shows her something on a touchpad. "I have to go now, Captain," she tells you. "Good luck."`
				decline
	on decline

> 		log "Asked Taely about the Graveyard and the Ka'het. She told me the previous contacts between them and the Remnant."

Reviewer: Can't use "me" in logs.
```suggestion
		log "Asked Taely about the Graveyard and the Ka'het. She spoke of the previous contacts between them and the Remnant."
```
Maybe expand this log to also mention the technology that the Remnant gained from salvaging the Ka'het shells.

Author: Well, the player won't necessarily know about the tech they retrieved, since it's an optional choice

> 		log "Factions" "Ka'het" `The Ka'het are aliens which live in an area south of the Ember Waste, called Graveyard, and attack everyone on sight. They are large lifeforms which can travel in space using artificial exoskeletons.`

Reviewer: ```suggestion
		log "Factions" "Ka'het" `The Ka'het are aliens which live in an area south of the Ember Waste, called the Graveyard. They are large lifeforms which can travel in space using artificial exoskeletons and will attack anyone on sight, even sometimes other Ka'het.`
```

> mission "First Contact: Ka'het: Quarg 1A"

Reviewer: How about instead of having a number of missions that account for the various Quarg that the player may or may not know (which would need to be expanded upon as new Quarg locations are added), how about you just have a single mission that tells you to find a Quarg to speak to, and the player can then land on any Quarg location to get their side of the story, then that mission speaking with the Quarg fails this one. It would be far simpler than what you're trying to do here. (Think of it like when you first contact the Wanderers after also having visited the Hai. That mission doesn't point you directly to Hai-home, it just tells you in general to look for a Hai to help you.)

Author: But are you sure that would solve the issue I currently have of the mission failing the *next* landing instead of when you do the second mission? In the meanwhile, I will change it to directly point the player to Lagrange (e.g. closest quarg station) and directly remove the rest

	name "Abandoned Ringworld"
	description "Visit a Quarg station to ask them about the abandoned ring in the middle of Ka'het space."
	landing
	source "Querii Ka"
	destination "Lagrange"
	to offer
		not "First Contact: Ka'het: Quarg 1B: offered"
		not "First Contact: Ka'het: Quarg 1C: offered"
		not "First Contact: Ka'het: Quarg 1D: offered"
		has "First Contact: Quarg: offered"
		not "First Contact: Hai: offered"
		not "First Contact: Kuwaru Efreti: offered"
	to complete
		never
	on offer

> 		log "Discovered an uninhabited, complete Quarg ringworld in the middle of Ka'het space."

Reviewer: ```suggestion
		log "Discovered a complete but uninhabited Quarg ringworld in the middle of the Graveyard."
```
Perhaps expand this log to say something along the lines of "It seems strange that the Quarg would abandon a ringworld given how important they are to them."

			`Due to your past experience, you expect crowds of the tall and inexpressive Quarg walking near the landing pads of this ringworld; instead, the whole station seems uninhabited.`
			`	Apart from the utter silence, which is pretty common in most rings, you are convinced by the fact that every room, apartment and cabin has been carefully emptied, leaving nothing behind. Perhaps it would be wise to ask the Quarg what happened here...`
				accept

mission "First Contact: Ka'het: Quarg 2"
	landing
	invisible
	source
		planet "Alta Hai" "Lagrange" "Kuwaru Efreti"
	to offer
		or
			has "First Contact: Ka'het: Quarg 1A: active"
			has "First Contact: Ka'het: Quarg 1B: active"
			has "First Contact: Ka'het: Quarg 1C: active"
			has "First Contact: Ka'het: Quarg 1D: active"
	on offer
		conversation
			`Do you want to ask the Quarg about the abandoned ringworld in the region of the Graveyard?`
			choice
				`	(Yes.)`
				`	(Not now.)`
					defer
			`	Walking down the principal, alien-looking hall, you decide to stop one of the few Quarg you see there. "Hello, human. How may I be of service?" the Quarg says. You reply by asking him about the Graveyard and the ringworld they built there.`

> 			`	"Ah, you encountered the remainders of the Builders. Millennia ago, they were a flourish and thoughtful empire, but hostile contacts forced them to grow fearful and hawkish. Although they never did attack us, they always refrained from accepting our aid, and attempted to relocate at a greater distance from us, in worlds easier to maintain than the home they were reserved."`

Reviewer: "The home they were reserved"?

Author: changed

			choice
				`	"Who were the hostile contacts with?"`
				`	"What happened to the Builders?"`
					goto ka'het

> 			`	"Long ago, few ships coming from a foreign species had reached that region of space, using some of the wormholes leading there. They belonged to a young and foolish race, and they foolishly decided to attack the Builders instead of learning from them. They were stronger any vessel the Builders possessed, but also few enough to be destroyed by the fleets patrolling the area."`

Reviewer: Repetitive use of "foolish".
```suggestion
			`	"Long ago, few ships coming from a foreign species had reached that region of space, using some of the wormholes leading there. They belonged to a young race who foolishly decided to attack the Builders instead of learning from them. Their ships were stronger than any vessel the Builders possessed at the time, but were also few enough in number to be destroyed by the Builders."`
```

			choice
				`	"What happened after that?"`
			label ka'het

> 			`	The Quarg stops for a few seconds before continuing, as if he was reflecting on something. "For many millennia they grew, developing brilliant pieces of technology. Then, compelled by their past experiences, they began creating weapons not only to defend themselves, but also to retaliate; these 'Ka'het', as they named them, defied shortly after they were complete.`

Reviewer: Defied what?
```suggestion
			`	The Quarg stops for a few seconds before continuing, as if he was reflecting on something. "For many millennia they grew, developing brilliant pieces of technology. Then, compelled by their past experiences, they began creating weapons not only to defend themselves, but also to retaliate; these 'Ka'het', as they named them, turned on their creators shortly after they were complete.`
```

> 			`"For few more centuries the Builders lasted, before falling under their very own hubris. Their space is still populated by the Ka'het their gave birth to through the immense facilities they assembled, who do not yet possess a vast intelligence." After saluting you, the Quarg quickly walks away, leaving you with more questions than answers.`

Reviewer: Saluting? The Quarg bow.
```suggestion
			`	"For few more centuries the Builders lasted, before falling under their very own hubris. Their space is still populated by the Ka'het they gave birth to through the immense facilities they assembled, who do not yet possess a vast intelligence." After bowing, the Quarg quickly walks away, leaving you with more questions than answers.`
```

				decline
	on decline
		log "Asked the Quarg about the history of the Graveyard. Although not being told the reason for the abandoned ringworld, have learned about the previous inhabitants of the region, the Builders."

> 		log "Factions" "Builders" `The Builders inhabited the Graveyard millennia ago, but have become extinct since. They developed some sort of weapon, called the Ka'het, which stopped obeying their orders and rebelled against them.`

Reviewer: ```suggestion
		log "Factions" "Builders" `The Builders inhabited the Graveyard millennia ago, but have since become extinct. They developed some sort of weapon, called the Ka'het, which stopped obeying their orders and rebelled against them.`
```

		has "First Contact: Kuwaru Efreti: offered"
	to fail
		has "First Contact: Ka'het: Quarg 2: offered"
	on offer
		log "Discovered an uninhabited, complete Quarg ringworld in the middle of Ka'het space."
		conversation
			`Due to your past experience, you expected to crowds of the tall and inexpressive Quarg walking near the landing pads of this ringworld; instead, the whole station seems uninhabited.`
			`	More than the utter silence, which is pretty common in most rings, you are convinced by the fact that every room, apartment and cabin has been carefully emptied, leaving nothing behind. Perhaps it would be wise to go ask the Quarg what happened here...`
				accept

mission "First Contact: Ka'het: Quarg 2"
	landing
	invisible
	source
		planet "Alta Hai" "Lagrange" "Kuwaru Efreti"
	to offer
		or
			has "First Contact: Ka'het: Quarg 1A: active"
			has "First Contact: Ka'het: Quarg 1B: active"
			has "First Contact: Ka'het: Quarg 1C: active"
			has "First Contact: Ka'het: Quarg 1D: active"
	on offer
		conversation
			`Do you want to ask the Quarg about their abandoned ringworld in the Graveyard?`
			choice
				`	(Yes.)`
				`	(Not now.)`
					defer

> 			`	Walking down the principal, alien-looking hall, you decide to stop one of the few Quarg you see there. "Hello, human. What help do you need?" he says, and you reply asking him what happened in the Graveyard and to the ringworld they built there.`

Reviewer 2: `	Walking down the principal, alien-looking halls, you decide to stop one of the few Quargs you see there. "Hello, human. What help do you need?" the Quarg says. You reply by asking him about the Graveyard and the ringworld they built there.`

> 			`	"Ah, you encountered the remainder of the Builders. Millennia ago, they were a flourish and thoughtful empire, but hostile contacts forced them to grow fearful and hawkish. Although they never did attack us, they refrained from accepting our aiding, and attempted to relocate at a greater distance from us."`

Reviewer 2: `	"Ah, you encountered the remains of the Builders. Millennia ago, they were a flourish and thoughtful empire, but hostile contacts forced them to grow fearful and hawkish. Although they never did attack us, they refrained from accepting our aid, even going so far as to attempt to relocate at a greater distance from us."`

Author: In my opinion the word "remainders" fits the Quarg dialogs better

			choice

> 				`	"Who were the hostile contacts with?"`

Reviewer 2: `	"Who were these hostile contacts?"`

Author: I'm not sure this is better than what we currently have

Reviewer: Doesn't seem like an entirely necessary change, but I do somewhat prefer this suggestion.

Author: Alright, changed

> 				`	"What happened to these Builders?"`

Reviewer 2: `	"What happened to the Builders?"`

					goto ka'het

> 			`	"Few ships coming from a foreign species had reached that region of space, using some of the wormholes leading there. They belonged to a young and foolish race, and they foolishly decided to attack the Builders instead of learning from them. They were stronger any vessel the Builders possessed, but also few enough to be destroyed by the Builders' fleets."`

Reviewer 2: `	"A long time ago, some ships coming from these foreign species had reached that region of space, using some of the wormholes leading there. They belonged to a young and foolish race who decided to attack the Builders instead of learning from them. They had stronger vessel than any the Builders possessed, but also few enough to be destroyed by the Builders' fleets."`

			choice

> 				`	"What happened to these Builders after that?"`

Reviewer 2: `	"What happened to the Builders after that?"`

			label ka'het

> 			`	The Quarg stops for a few seconds before continuing, as if he was reflecting on something. "For many millennia they grew, developing brilliant pieces of technology. Then, compelled by their past experiences, they began creating weapons not only to defend themselves, but also to retaliate; these 'Ka'het', as they named them, defied shortly after they were complete." After saluting you, the Quarg slowly walks away, leaving you with more questions than answers.`

Reviewer 2: `	The Quarg stops for a few seconds before continuing, as if reflecting on something. "For many millennia they grew, developing brilliant pieces of technology. Then, compelled by their past experiences, they began creating weapons not only to defend themselves, but also to retaliate; these 'Ka'het', as they named them, defied the Builders shortly after they were complete." After saluting you, the Quarg slowly walks away, leaving you with more questions than answers.`

> 			`	The Quarg stops for a few seconds before continuing, as if he was reflecting on something. "For many millennia they grew, developing brilliant pieces of technology. Then, compelled by their past experiences, they began creating weapons not only to defend themselves, but also to retaliate; these 'Ka'het', as they named them, defied shortly after they were complete." After saluting you, the Quarg slowly walks away, leaving you with more questions than answers.`

Reviewer 2: > After saluting you, the Quarg slowly walks away, leaving you with more questions than answers.`
I've reservations on this. Like why would the Quarg salute you and just leave?

Author: That was the best I could think of to make the Quarg equivalent of ending a conversation (seeing how few the Quarg said about the sheragi, it's already incredible the convo lasted that long); if that is not good enough, any suggestions on what should I use instead?

	on decline

> 		log "Asked the Quarg about the history of the Graveyard. Although I wasn't told the reason for the abandoned ringworld, I have learned about the previous inhabitants of the region, the Builders."

Reviewer 2: log "Asked the Quarg about the history of the Graveyard. Although not being told the reason for the abandoned ringworld, have learned about the previous inhabitants of the region, the Builders."

			choice
				`	"What happened to these Builders after that?"`
			label ka'het
			`	The Quarg stops for a few seconds before continuing, as if he was reflecting on something. "For many millennia they grew, developing brilliant pieces of technology. Then, compelled by their past experiences, they began creating weapons not only to defend themselves, but also to retaliate; these 'Ka'het', as they named them, defied shortly after they were complete.`
			`"For few more centuries the Builders lasted, before falling under their very own hubris. Their space is still populated by the Ka'het their gave birth to through the immense facilities they assembled, who do not yet possess a vast intelligence." After saluting you, the Quarg quickly walks away, leaving you with more questions than answers.`
	on decline
		log "Asked the Quarg about the history of the Graveyard. Although I wasn't told the exact reason for the abandoned ringworld, I have learned about the previous inhabitants of the region, the Builders."

> 		log "Factions" "Builders" `The Builders inhabited the Graveyard thousands of years ago, but have fallen extinct long ago. They developed some sort of weapon, called Ka'het, which stopped obeying their orders and insurged against them.`

Reviewer 2: log "Factions" "Builders" `The Builders inhabited the Graveyard millennials ago, but have become extinct since. They developed some sort of weapon, called the Ka'het, which stopped obeying their orders and rebelled against them.`

> 		log "Factions" "Ka'het"  `According to the Quarg, the Ka'het were initially a weapon created by the Builders, which then defied and led them to extinction. Even if millennia passed, they are still being produced in the Graveyard.`

Reviewer 2: log "Factions" "Ka'het"  `According to the Quarg, the Ka'het were initially a weapon created by the Builders, which then rebelled and led them to extinction. Even if millennia passed, they are still being produced in the Graveyard.`

Author: What's the point in changing defied with rebelled?

Reviewer: "Defied" does sound strange to me. You're using this in a similar way to the sentence "The Ka'het defied," to which I ask "defied what?" "Defied" usually doesn't just sit alone like that.

Reviewer: Still have a strange use of "defied" here.
